controllers/component_build_controller_test.go

```diff
@@ -702,10 +708,23 @@ var _ = Describe("Component initial build controller", func() {
 			serviceAccount.ImagePullSecrets = []corev1.LocalObjectReference{{Name: generatedImageRepoSecretName}, {Name: anotherSecretName}}
 			Expect(k8sClient.Update(ctx, serviceAccount)).Should(Succeed())
 
+			generatedImageRepoSecretKey := types.NamespacedName{Namespace: resourcePacPrepKey.Namespace, Name: generatedImageRepoSecretName}
+			// wait for secret to be removed because component was removed in afterEach
+			waitSecretGone(generatedImageRepoSecretKey)
 			assertLinkSecretToSa(true)
 
 			// will remove just generate image repo secret from SA and keep anothersecret
 			deleteComponent(resourcePacPrepKey)
+			// wait for secret to be removed
```

Comment:
nitpick: this comment duplicates the following method name, I'd remove it.

Reply:
ok